### PR TITLE
Gw crawler

### PR DESCRIPTION
**Pull request by samarthdd**
implemented scrapy,
created protect structure with base class, config file to configure scrape sites,
used file pipeline to download zip file,
Created a docker image for whole process.

### CODE REVIEW

File: upwork-devs/samarth_dd/gw_crawler/malicious_file_crawler/src/config/config.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[SCRAPE_SITE]
+scrape_sites = CORVUS
+
+[CORVUS]
+name = glasswall_spider
+login_url = https://cloud.corvusforensics.com/s/HDcCHLnQTGBnYBN
```
Comment (DinisCruz) [inline upwork-devs/samarth_dd/gw_crawler/malicious_file_crawler/src/config/config.ini:6]: Is this an API key?

File: upwork-devs/samarth_dd/gw_crawler/malicious_file_crawler/src/spiders/glasswall_crawler.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+""" Scraper class for getting vehicles from carwave portal """
```
Comment (DinisCruz) [inline upwork-devs/samarth_dd/gw_crawler/malicious_file_crawler/src/spiders/glasswall_crawler.py:2]: Carwave?

File: upwork-devs/samarth_dd/gw_crawler/malicious_file_crawler/src/spiders/glasswall_crawler.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import scrapy
+import wget
+from scrapy.loader import ItemLoader
+from malicious_file_crawler.src.items import MaliciousFileCrawlerItem
+import json
+import requests
+from malicious_file_crawler.src.spiders.scraper import Scraper
+from selenium import webdriver
```
Comment (DinisCruz) [inline upwork-devs/samarth_dd/gw_crawler/malicious_file_crawler/src/spiders/glasswall_crawler.py:10]: We should be using pyppeteer

**Review by DinisCruz — APPROVED** [review]

**Review by susilupwork — COMMENTED** [review]
- Add Unit test
- Follow glasswall coding guideline. 
- Implement comments given to Ashish.

**Pull request merged**